### DISCUSSION TITLE
How can I use this with a Python script?

### DISCUSSION THREAD

**Discussion by nicolello-dev** [Tyrrrz/DiscordChatExporter#1105] [Q&A]
As the title says, I would like to use the chat exporter with Python. Is there a way other than the CLI to accomplish this?

**Comment by CanePlayz** [comment]
It's pretty easy to send lines to the terminal with Python

**Reply by nicolello-dev** [reply]
Of course; I just wanted to know if someone made a library to work with it without resorting to the terminal

**Reply by CanePlayz** [reply]
Not to my knowledge
